lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -88,7 +90,7 @@ public function query(string $sql) : ResultStatement
      */
     public function quote($input, $type = ParameterType::STRING)
     {
-        return $this->connection->quote($input, $type);
+        return $this->connection->quote((string) $input, $type);
```

Comment:
Wait, PDO doesn't specify a type for `$input`? :|

Reply:
It comes as mixed in here, but quote() expects string: http://php.net/manual/en/pdo.quote.php

Reply:
Thinking out loud: should we restrict it to `string` as well? What's the point in quoting/escaping anything else than a string?

Reply:
I think it makes sense, but not in this PR - in #3348.